### ISSUE TITLE
Test suite seems to be broken

### ISSUE THREAD

**Issue by jeroenvandijk** [caelum/hypertemplate#2]
I did a `bundle install` and after that a `bundle exec rake` and that didn't work. Then I changed the gemspec a bit to get all the dependencies, but it seems there is a conflict between the test Rails app that requires Rails 2.3.8 with a dependency on Rack version 1.1.0 whereas the Rack version required by the Gemfile is ~> 1.2.0...

Can you confirm this?

I wanted to do a patch for Restfulie to add MultiJson instead of json_pure

**Comment by guilhermesilveira** [comment]
You are correct. there is a conflict between the test app and the gem itself... if you can update the test app to any other compatible version, go for it! ill take a look at the patch now

**Comment by kshahkshah** [comment]
I've seen the same, and though I'm using Rails 2.3, I'm almost tempted to just settle on testing for Rails 3 only.

Maybe drop support for the old DSL?

Thoughts?
